sycl/test/regression/macro_conflict.cpp

```diff
@@ -0,0 +1,25 @@
+// RUN: %clangxx -fsycl %s -o %t.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
```

Comment:
I don't think it makes a lot of sense to launch this test, especially on accelearator

Reply:
Sure, I will remove it.